src/Report/Xml/Source.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types = 1);
+/*
+ * This file is part of the php-code-coverage package.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace SebastianBergmann\CodeCoverage\Report\Xml;
+
+use TheSeer\Tokenizer\NamespaceUri;
+use TheSeer\Tokenizer\Tokenizer;
+use TheSeer\Tokenizer\XMLSerializer;
+
+class Source
+{
+    /** @var \DOMElement */
+    private $context;
+
+    /**
+     * @param \DOMElement $context
```

Comment:
With PHP 7.0+, PHPDoc pretty much never adds any technical value. Yet, it is still considered a good practice to have them.